test/views/list-item/content/static.test.js

```diff
@@ -8,7 +8,7 @@ const ContentViewStatic = require('../../../../lib/views/list-item/content/stati
 const assert = require('assert');
 const enzyme = require('enzyme');
 const sinon = require('sinon');
-const React = require('react');
+const React = require('react'); // eslint-disable-line no-unused-vars
```

Comment:
I think `React` is required to be in scope for when JSX is transpiled. Do we need this `eslint-disable-line` though? I thought the eslint react plugin would be smart about this.